linera-core/src/client.rs

```diff
@@ -1865,6 +1878,55 @@ where
         }
     }
 
+    /// Changes the ownership of this chain. Fails if it would remove existing owners, unless
+    /// `remove_owners` is `true`.
+    pub async fn change_ownership(
+        &mut self,
+        ownership: ChainOwnership,
+        remove_owners: bool,
```

Comment:
What is the motivation for this parameter?

Reply:
I removed it, and the new error type.